loader.js

```diff
@@ -1276,6 +1385,103 @@ var Module = null;
      getfullscreenenabler().call(this._canvas);
    };
 
+   /*
+    * V86Runner
+    */
+   function V86Runner(canvas, game_data) {
+     // v86 needs a specific DOM structure instead of a canvas
+     var screenContainerOuterElt = document.createElement("div");
+     screenContainerOuterElt.id = canvas.id;
+     screenContainerOuterElt.classList = canvas.classList;
+     screenContainerOuterElt.style = canvas.style;
+
+     var screenContainerInnerElt = document.createElement("div");
+     screenContainerInnerElt.classList = ["emularity-v86-screen-container"];
+     screenContainerInnerElt.style = "display:flex;justify-content:center;align-items:center;background-color:#000;";
+
+     var textDivElt = document.createElement("div");
+     textDivElt.classList = ["emularity-v86-screen-text"];
+     textDivElt.style = "font-size:14px;font-family:monospace;line-height:14px;white-space:pre;";
+     var canvasElt = document.createElement("canvas");
+     canvasElt.classList = ["emularity-v86-screen-canvas"];
+     canvasElt.style = "display:none;";
+
+     screenContainerInnerElt.appendChild(textDivElt);
+     screenContainerInnerElt.appendChild(canvasElt);
+     screenContainerOuterElt.appendChild(screenContainerInnerElt);
+     canvas.parentNode.replaceChild(screenContainerOuterElt, canvas);
+
+     var cfg = {};
+     cfg.screen_container = screenContainerInnerElt;
+     cfg.memory_size = game_data.memory_size || 32 << 20;
+     cfg.vga_memory_size = game_data.vga_memory_size || 2 << 20;
```

Comment:
I would move these manipulations into the V86Loader methods.

Reply:
I've based this convention off SAELoader/SAERunner, which uses manipulations in Runner likewise. I've changed it to an alternate approach for setting defaults, which seems to work fine; is this more in line with what you'd expect?